include/spm/evt_sub.h

```diff
@@ -17,6 +17,10 @@ EVT_DECLARE_USER_FUNC(evt_sub_random, 2)
 EVT_DECLARE_USER_FUNC(func_800d4de4, 2)
 EVT_DECLARE_USER_FUNC(func_800d74a0, 2)
 
+// Returns the door/bero name
+// evt_sub_get_entername(&int ret)
```

Comment:
This should be a reference to a char * rather than to an int

Reply:
right; I was thinking of LW as an int instead of the door name as char*. Will fix in next commit.